### PR TITLE
843 v170 feedbacks 2

### PR DESCRIPTION
**Pull request by ismaelAbogabal**
# Issues 
1. home screen salah time sizes are tiny 
2. the top margin in the secondary screens 
3. Magrib size is a bit smaller than other 
4. Mini salah bar sizes are small in the Arabic language 
5. flash messages some times stuck

### CODE REVIEW

File: lib/src/pages/home/widgets/AboveSalahBar.dart [hunk 1]
```diff
@@ -54,46 +49,58 @@ class AboveSalahBar extends StatelessWidget {
...
+                //                 shadows: kHomeTextShadow,
+                //                 letterSpacing: 1.vr,
+                //                 height: .9,
+                //                 color: Colors.white,
+                //                 fontWeight: FontWeight.w500,
+                //               ),
+                //         ),
+                //       ),
+                //   ],
+                // ),
```
Comment (ibrahim-zehhaf-mawaqit) [inline lib/src/pages/home/widgets/AboveSalahBar.dart:103]: don't commit commented code please

File: lib/src/pages/home/widgets/salah_items/responsive_mini_salah_bar_widget.dart [hunk 2]
```diff
@@ -85,33 +85,33 @@ class ResponsiveMiniSalahBarWidget extends StatelessOrientationWidget {
...
+              //   child: MiniHorizontalSalahItem(
+              //     title: mosqueProvider.salahName(4),
+              //     time: todayTimes[4],
+              //     active: nextActiveIqama == 4,
+              //   )
+              //       .animate(delay: _step * 4)
+              //       .fadeIn(duration: _duration)
+              //       .slideY(begin: 1, duration: _duration, curve: Curves.easeOut)
+              //       .addRepaintBoundary(),
+              // ),
```
Comment (ibrahim-zehhaf-mawaqit) [inline lib/src/pages/home/widgets/salah_items/responsive_mini_salah_bar_widget.dart:114]: don't commit commented code please

File: lib/src/widgets/time_widget.dart [hunk 3]
```diff
@@ -65,7 +68,7 @@ class TimeWidget extends StatelessWidget {
         SizedBox(width: 1.vw),
         TimePeriodWidget(
           dateTime: time!.toDate(),
-          style: amPmStyle ?? style?.apply(color: Colors.grey.shade300, fontSizeFactor: 0.5),
+          style: amPmStyle ?? style?.apply(color: Colors.grey.shade300, fontSizeFactor: isArabicLang ? .5 : 0.5),
```
Comment (ibrahim-zehhaf-mawaqit) [inline lib/src/widgets/time_widget.dart:71]: .5 and 0.5 are not the same thing ?

**Comment by ismaelAbogabal** [comment]
@ibrahim-zehhaf-mawaqit removed the commented code. please check it out.

**Review by ibrahim-zehhaf-mawaqit — APPROVED** [review]

**Pull request merged**